index.js

```diff
@@ -59,16 +59,16 @@ const generateDbShopLink = async (outbound, opt) => {
 	}
 
 	const orig = outbound.legs[0].origin
-	const originId = orig.station && orig.station.id || orig.id || orig
+	const originId = orig.station?.id || orig.id || orig
```

Comment:
This raises the minimum Node.js version to 14.x, but currently it is 10.x. Therefore, this is a breaking change (which is fine).

Reply:
Didn't even think about that, to be honest. Just made it easier for me to read and understand. I can change it back or leave it as is, either way is fine with me.